app/src/main/java/com/CodeNaroNa/vendor/relief/Deprecated/Activity/CoronaSpreadData.java

```diff
@@ -1,4 +1,4 @@
-package com.CodeNaroNa.vendor.relief.Activity;
+package com.CodeNaroNa.vendor.relief.Deprecated.Activity;
 
```

Comment:
Instead of keeping java class delete them

Reply:
Alright